card/mod/01_core/set/all/templating.rb

```diff
@@ -82,7 +82,7 @@ def update_structurees args
   # note that this is not smart about overriding templating rules
   # for example, if someone were to change the type of a +*right+*structure rule that was overridden
   # by a +*type plus right+*structure rule, the override would not be respected.
-  if query = structuree_spec
+  if (query = structuree_spec)
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.